data/persons.txt

```diff
@@ -1850,3 +1849,158 @@ ship "Modified Ladybug"
+
+
+person "Patrol Team"
+	government "Author"
+	frequency 300
+	personality
+		mining harvests plunders
+	phrase
+		word
+#		Marksman
+			"If you don't know where to start in helping the game, take a nice look at the files. Sometimes, it helps!"
+			"If you're interested in helping out but don't have time, consider being a proofreader! You can take my job, and I get to laze around doing nothing!"
+			"If you see something, and it shoots at you, cheat your way to victory!"
+			"What did you just say about me?"
+			"The number of times someone breaks the fourth wall, it's mind-boggling!"
+			"Look, we aren't here for your money. We do all this for free and look where that gets us."
+			"If you don't know where to start in helping the game, come and review some missions on GitHub. It helps!"
+			"If you're interested in helping out but don't have time, consider looking through the proposals on GitHub! Sometimes, a new perspective is needed!"
+			"What do you mean you don't have a save file for this pull request!?"
+			"Please, for the love of god, don't make an issue before checking if others have done the same."
+			"This statement is, intentionally wrote wrongly to make point."
+
+#		Roadrunner
+			"Typo detected. Initiating weapons engagement."
+			"Code non-functional. Destroy them."
+
+#		Saugia
+			"Yea... way too much detail. I can't see the cockpit anymore."
+			"Just don't go ham on details."
+			"These areas are looking much better, though. The contrast of the lesser details here and the more elsewhere is a lot nicer."
+			"I'm just trying to see this half that size and I'm losing a lot of detail."
+			"It's awesome, but far too detailed."
+			"Now there's a lot less detail conflicting with one another."
+
+#		tibetiroka
+			"Lol, we broke master again."
+			"It's just the Steam CI, no need to panic."
+			"DERPY???!!!!!"
+			"Did you check for nullptr?"
+			"		How do we indent after line breaks again?"
+			"Please put the includes in alphabetical order."
+			"Uhh, you sure about this?"
+			"Always keep three empty lines between functions."
+			"aLiEnS DoN'T SpEaK EnGlIsH, nO NeEd tO ChEcK ThE GrAmMaR,"
+
+#		1010todd
+			"Style check."
+			"Quick pass."
+			"I'll give this a better look later."
+			"What is this for?"
+			"This should be a private member."
+			"Comments?"
+			"Please remove unnecessary space."
+
+			"Looks good to me."
+			"Is this the best name?"
+			"<reviewer> requested a review from (Amazinite/Tehhowch)"
+			"The standard copyright header was changed recently."
+
+			"Please use the template."
+			"The decision to implement fancy quote is one of the worst in computer history."
+			"	This needs another indent."
+#		EjoThims
```

Comment:
iirc I wrote these on the doc, they're based on inspiration from warp-core, and they represent Leaf mostly so I'd just write it's code things?
```suggestion
#		Code
```

Reply:
That depends on the direction we want to go with this. Saugia raised a good point on having maybe 4-8 per person, so we could add more for new reviewers, but we'll see.

Reply:
thing is this isnt really how Ejo would speak? idk its just a bit weird to me, this could just represent code ppl without being specific as to who (bc these arent really things a mission reviewer would say?) then again it's just a comment
~~dont need to count me in bc I still hope for a uhai pers ship after uhai gets merged~~

Reply:
I thought that it was just crediting who the idea for each line came from, not being attributed directly to anyone specific when said in game?

I do recall listing at least a few things similar to the ones in my section when ideas were asked for, but even then I was thinking in more general terms, not how I specifically would say things.

Reply:
~~reviewing code PRs now are you Ejo~~
I mean no problem as far as I'm concerned

Reply:
Looking at the doc, the main one was mainly from fancy quote related, or something about there being too many commas, so...

Reply:
> ~~reviewing code PRs now are you Ejo~~

Gotta check that comment grammar ;)

Reply:
> Looking at the doc, the main one was mainly from fancy quote related, or something about there being too many commas, so...

Yea, those both ring a bell, too lol

So is each list supposed to be what's coming from that reviewer in game, or is it just listing contributions towards the pool of things any reviewer ship may say?

I'm now confused on what is actually intended here :3

Reply:
> > Looking at the doc, the main one was mainly from fancy quote related, or something about there being too many commas, so...
> 
> Yea, those both ring a bell, too lol
> 
> So is each list supposed to be what's coming from that reviewer in game, or is it just listing contributions towards the pool of things any reviewer ship may say?
> 
> I'm now confused on what is actually intended here :3

I think what happened last year when I was still making this was that it was intended to be for what each ship would say, but now it's been turned to what each reviewer had said before.

This means Todd's statement will, unfortunately, have to be voided. But we'll get to it when we get to it.

Reply:
My quotes are also not necessarily what I say, just ideas I came up with.

Reply:
It may make sense to have comments indicating who is being quoted, or who the phrase represents, but I don't think it make sense to have comments indicating who added the lines. We don't do this anywhere else beyond the copyright header when an individual his responsible for very large parts of the content of a file.